jax/image/scale.py

```diff
@@ -115,14 +115,31 @@ def _scale_and_translate(x, output_shape, scale, translate, kernel,
   return jnp.einsum(x, in_indices, *contractions, precision=precision)
 
 
+def _resize_nearest(x, output_shape):
+  input_shape = x.shape
+  assert len(input_shape) == len(output_shape)
+  spatial_dims = np.nonzero(np.not_equal(input_shape, output_shape))[0]
```

Comment:
nit: I found tuple unpacking a bit more readable for these cases:
```suggestion
  spatial_dims, = np.nonzero(np.not_equal(input_shape, output_shape))
```

Reply:
Done. I also did the same in the non-nearest-neighbor case.